doc/project/docker.rst

```diff
@@ -21,18 +21,18 @@ building of `Docker`_ containers. PDAL keeps three Docker containers current.
 
 .. note::
 
-    Containers are built upon the `Dependences`_ container, but the
-    `Dependences`_ container is not pinned to specific Xenial or PDAL release
-    times. It corresponds to where ever the ``dependencies`` tag of the PDAL
+    PDAL's public use Docker containers are built using `Alpine Linux <https://alpinelinux.org/>`__,
+    but the containers that PDAL's `Travis`_ infrastructure uses
+    are built upon Ubuntu. The `Dependences`_ container
```

Comment:
Are you changing this separately? Pretty sure Travis currently does use Alpine, even in the PR.

Reply:
oh, I hadn't looked. If Travis is running Alpine, that means we haven't been running the Java stuff for a while.